package com.api.socialNetwork.exception

class RelationAlreadyExistisException : RuntimeException("This relation already exists")